rpmlint/lint.py

```diff
@@ -83,30 +84,34 @@ def _load_installed_rpms(self, packages):
                 print_warning(f'(none): E: there is no installed rpm "{name}".')
         return existing_packages
 
+    def _find_rpmlintrc_files(self, path):
+        rpmlintrcs = []
+        rpmlintrcs += sorted(path.glob('*.rpmlintrc'))
+        rpmlintrcs += sorted(path.glob('*-rpmlintrc'))
+        return rpmlintrcs
+
     def _load_rpmlintrc(self):
         """
         Load rpmlintrc from argument or load up from folder
         """
-        if self.options['rpmlintrc']:
-            self.config.load_rpmlintrc(self.options['rpmlintrc'])
-        else:
-            # load only from the same folder specname.rpmlintrc or specname-rpmlintrc
-            # do this only in a case where there is one folder parameter or one file
-            # to avoid multiple folders handling
-            rpmlintrc = []
-            if not len(self.options['rpmfile']) == 1:
-                return
-            pkg = self.options['rpmfile'][0]
-            if pkg.is_file():
-                pkg = pkg.parent
-            rpmlintrc += sorted(pkg.glob('*.rpmlintrc'))
-            rpmlintrc += sorted(pkg.glob('*-rpmlintrc'))
-            if len(rpmlintrc) > 1:
-                # multiple rpmlintrcs are highly undesirable
-                print_warning('There are multiple items to be loaded for rpmlintrc, ignoring them: {}.'.format(' '.join(map(str, rpmlintrc))))
-            elif len(rpmlintrc) == 1:
-                self.options['rpmlintrc'] = rpmlintrc[0]
-                self.config.load_rpmlintrc(rpmlintrc[0])
+        if not self.options['rpmlintrc']:
+            # first load SUSE-specific location
+            sources = Path('/home/abuild/rpmbuild/SOURCES')
```

Comment:
You don't need the other location?

Reply:
Do you want me to drop the following:
```
-                # load only from the same folder specname.rpmlintrc or specname-rpmlintrc
-                # do this only in a case where there is one folder parameter or one file
-                # to avoid multiple folders handling
-                pkg = self.options['rpmfile'][0]
-                if pkg.is_file():
-                    pkg = pkg.parent
-                self.options['rpmlintrc'] += self._find_rpmlintrc_files(pkg)
```

Reply:
No I thought that you had the glob for 2 locations in description and here you use just one, otherwise it is good enough.